src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -843,6 +847,52 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
     });
   }
 
+  /**
+   * Reset SSH Keypair input field (private/public)
+   */
+  _initManualSSHKeypairFormDialog() {
+    this.enteredSSHPublicKeyInput.value = '';
+    this.enteredSSHPrivateKeyInput.value = '';
+  }
+
+  /**
+   * Open SSH Keypair Form dialog
+   */
+  _openSSHKeypairFormDialog() {
+    this.sshKeypairFormDialog.show();
+  }
+
+  /**
+   * Hide SSH Keypair Form dialog
+   */
+  _hideSSHKeypairFormDialog() {
+    this.sshKeypairFormDialog.hide();
+  }
+
+  /**
+   * Save SSH Keypair Form dialog
+   */
+  _saveSSHKeypairFormDialog() {
+    const sshPublicKey = this.enteredSSHPublicKeyInput.value;
+    const sshPrivateKey = this.enteredSSHPrivateKeyInput.value;
+    const p = globalThis.backendaiclient.postSSHKeypair({
+      pubkey: sshPublicKey,
+      privkey: sshPrivateKey
+    });
+    p.then((resp) => {
+      this.notification.text = _text('usersettings.SSHKeypairEnterManuallyFinished');
+      this.notification.show();
+      this._hideSSHKeypairFormDialog();
+      this._openSSHKeypairRefreshDialog();
+    }).catch((err) => {
+      if (err && err.message) {
```

Comment:
Also, I think It's better to show the result of the ssh keypair registration operation request, regardless of the result (success or failure). So I just added a notification at the end of the request.

Reply:
👏